_drgn.pyi

```diff
@@ -529,6 +529,7 @@ class Program:
         tag: Optional[str],
         size: IntegerLike,
         members: Sequence[TypeMember],
+        template_parameters: Optional[Sequence[TypeTemplateParameter]] = None,
```

Comment:
What does it mean for `template_parameters` to be `None`? For `members`, there's a distinction: an empty sequence is no members, and `None` is an incomplete type. I don't see a distinction for `template_parameters`. Perhaps this should be `template_parameters: Sequence[TypeTemplateParameter] = ()` (also, `template_parameters` should be mentioned in the docstring).